README.md

```diff
@@ -64,9 +58,9 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+This is a project done during React module at [Microverse](https://microverse.org), a remote web-developer school, using their support and frameworks provided.
+For the data, I used [REST COUNTRIES](https://restcountries.com/) API.
+For the design, I used inpiration from [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio) using these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).
 
 ## 📝 License
```

Comment:
- [ ] [OPTIONAL] because the rest of your project looks good I will not reject it because of this, but please update it. the link to the license is not working, please have a look at [the Microverse template](https://github.com/microverseinc/readme-template) for an example of how it is supposed to work.

Reply:
Done, and thank you.